scroll-to-text-fragment/idlharness.window.js

```diff
@@ -0,0 +1,15 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+'use strict';
+
+idl_test(
+  ['scroll-to-text-fragment'],
+  ['html'],
+  idl_array => {
+    idl_array.add_objects({
+      Location: ['document.location'],
```

Comment:
I'm curious, as I've not been adding partial interfaces to the `idl_array` in some of the tests I've been working on. What additional tests does adding the partial interface here enable?

Reply:
It will check, in convoluted ways, if `document.location.fragmentDirective` is there, has the correct type, and so on. Without this only `Location.prototype.fragmentDirective` will be inspected.

It's rare that testing the instances turn up anything new if the interface object is exposed, though.